package.json

```diff
@@ -85,25 +90,16 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "yarn lint"
+      "pre-commit": "npm run lint"
     }
   },
-  "jest": {
-    "preset": "react-native",
-    "modulePathIgnorePatterns": [
-      "<rootDir>/example/node_modules",
-      "<rootDir>/lib/"
-    ],
-    "setupFilesAfterEnv": [
-      "@testing-library/jest-native/extend-expect"
-    ]
-  },
   "react-native-builder-bob": {
     "source": "src/internals",
     "output": "lib",
     "targets": [
       "commonjs",
-      "module"
+      "module",
+      "typescript"
```

Comment:
I think we don't want to let builder bob generate the type defs since we want to have the types be defined separately since it's practically impossible to get the type defs work correctly by inferring them from the internals.

Reply:
Oh, sorry this is my mistake.
I met test error at first after setting least configuration, and I thought this configuration needed. But, Only I needed was to rename index.ts to index.tsx to fix it. I'll remove it.